src/main/java/Duke.java

```diff
@@ -6,5 +11,44 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Scanner in = new Scanner(System.in);
+        String input;
+        ArrayList<String> items = new ArrayList<String>();
+        ArrayList<Boolean> marked = new ArrayList<Boolean>();
+        
+        while((input = in.nextLine()) != "") {
+            if (input.equalsIgnoreCase("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                in.close();
+                return;
+            }
+            else if (input.equals("list")) {
+                
+                for(int i = 0; i < items.size(); i++) {
+                    System.out.println(i+1 +". " +  (marked.get(i) ? "[X] " : "[ ] ")   + items.get(i));
+                    
+                }
+            }
+            else if (input.startsWith("mark")) {
+                String[] strArray = input.split(" ");
```

Comment:
I recommend executing the command String.trim() before using String.split(), as it would account for cases where the user has inputted whitespace(s) before the actual non-whitespace text.